Evergreen/MainWindow/MainWindowController.swift

```diff
@@ -69,6 +69,14 @@ class MainWindowController : NSWindowController, NSUserInterfaceValidations {
 		if item.action == #selector(openArticleInBrowser(_:)) {
 			return currentLink != nil
 		}
+        
+        if item.action == #selector(showShareWindow(_:)) {
+            if let link = currentLink {
+                return URL(string: currentLink) != nil
+            } else {
```

Comment:
Not sure how this `currentLink` use snuck in. My local copy has `link` being used.

I'm kind of torn on using `guard` when the statement in the scope is only a single line. I ended up changing to `guard` here and also changing to use `switch` for these cases, which I think better separates each selector path.